docs/src/dev-containers.md

```diff
@@ -0,0 +1,50 @@
+# Dev Containers
+
+Dev containers provide a consistent, reproducible development environment by defining your project's dependencies, tools, and settings in a container configuration.
+
+If your repository includes a `.devcontainer/devcontainer.json` file, Zed can open a project inside a development container.
+
+## Requirements
+
+- Docker (or a compatible CLI such as Podman) must be installed and available in your `PATH`.
```

Comment:
At this time, `docker` is a hard requirement. The user can `alias podman=docker` (or maybe podman does this by default?), but Zed does need `which docker` to evaluate.